.devcontainer/scripts/post-create.sh

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env bash
```

Comment:
Script name should be in sync with Python template (there: postCreateCommand.h)

Reply:
Done

Reply:
&#x1F44D;